### PR TITLE
add sniffer packet linux

### PR DESCRIPTION
**Pull request by remirobert**

### CODE REVIEW

File: libwire/inc/linuxpacketreader.h [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+ * 
+ */
+
+#ifndef LINUXPACKETREADER_H
+#define LINUXPACKETREADER_H
+
+#include <string.h>
+#include <iostream>
+#include <unistd.h>
+#include<sys/socket.h>
```
Comment (izissise) [inline libwire/inc/linuxpacketreader.h:24]: Does it work  without the space? :O

File: libwire/inc/IpHeader.h [hunk 2]
```diff
@@ -0,0 +1,73 @@
+/*
```
Comment (izissise) [inline libwire/inc/IpHeader.h:1]: Apache license ?

File: libwire/src/IpHeader.cpp [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+     
+		memset(&dest, 0, sizeof(dest));
+		dest.sin_addr.s_addr = ip_header->daddr;
+		
+		saddr = std::string(inet_ntoa(source.sin_addr));
+		daddr = std::string(inet_ntoa(dest.sin_addr));
+    }
+
+	IpHeader::~IpHeader() {}
+	bool IpHeader::operator==(const IpHeader& other) const { return true; }
```
Comment (izissise) [inline libwire/src/IpHeader.cpp:39]: Always true! Xd

**Review by izissise — APPROVED** [review]
Good job!!

File: libwire/inc/linuxpacketreader.h (hunk 1 above)
Comment (remirobert) [reply libwire/inc/linuxpacketreader.h:24]: Yes.

File: libwire/inc/IpHeader.h (hunk 2 above)
Comment (remirobert) [reply libwire/inc/IpHeader.h:1]: Kdevelop by default.

File: libwire/src/IpHeader.cpp (hunk 3 above)
Comment (remirobert) [reply libwire/src/IpHeader.cpp:39]: For now lol, I was thinking, about that. We must implement some filters.

**Pull request merged**